content/format-guide.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Format Guide for Facilators"
+date: 2020-01-14T19:02:06Z
+draft: true
+---
+
+One of the biggest differences with DeliveryConf from other events you have may have attended or volunteered at is our format. This format was conceived to encourage attendees to have discussions about how topics discussed affect their own organizations. After each talk there are 20 minutes set aside for discussion.
+
+##### Most important rule: These are discussions about the TOPIC, not the talk
+
+Other rules:
+
+- Stick to the time box. There are other tracks which will be starting on time. Don't make people late.
+- Nobody is allowed to speak if they aren't holding a microphone (even you). These discussions are being recorded at the sound board and will be distributed. Cross talk will make everything sound bad.
+- The speaker is not required to stay. They are more than welcome of course, but as stated this is meant to be a discussion about the topic, not the specific talk.
+
+Time Box:
+
+- 20 minutes - Split into roughly 3 equal parts. We know this isn't enough time to get deep, but there are lots of places in the venue for folks to continue discussions later. The goal here is to get discussion started while the topic is fresh.
+
+The questions:
+
+- What challenges are you facing in this area? This question is designed to let people ask for help.
+- What successes have you had in this area? There are things people have done in every topic area that would benefit the others.
+- If you had a magic wand, what would you want to see in 2-5 years? There will be many people in attendance who can influence the tools and platforms available to us. Let them know what we need.
+
+Running the session:
+
+- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the other should be handed to people who want to speak by the runner.
```

Comment:
Each room will have 2 hand held microphones. The facilitator should hold on to one of them, and the runner holds the other.